tests/boot/boot_from_pxe.pm

```diff
@@ -113,9 +113,9 @@ sub run {
         send_key "tab";
     }
     if (check_var('BACKEND', 'ipmi')) {
-        $image_path .= " ipv6.disable=1 "        if get_var('LINUX_BOOT_IPV6_DISABLE');
-        $image_path .= "ifcfg=$interface=dhcp4 " if (!get_var('NETWORK_INIT_PARAM') && !get_var('SUT_NETDEVICE_SKIPPED'));
-        $image_path .= 'plymouth.enable=0 ';
+        $image_path .= " ipv6.disable=1 "         if get_var('LINUX_BOOT_IPV6_DISABLE');
+	$image_path .= " ifcfg=$interface=dhcp4 " if (!get_var('NETWORK_INIT_PARAM') && !get_var('SUT_NETDEVICE_SKIPPED'));
+        $image_path .= ' plymouth.enable=0 ';
```

Comment:
this approach can add double paths. I recommend for any latter option to always prepend a space, e.g.

```
$image_path .= ' ipv6.disable=1' if …;
$image_path .= " ifcfg=$interface=dhcp4" if …;
$image_path .= ' plymouth.enable=0';
```

btw if `plymouth.enable=0` is generally required for PXE I would consider this a product bug which should at least tracked with a `record_soft_failure` unless this is something specific to our PXE setup

Reply:
I agree with "prepend a space" anyway. But not quite sure what is your "double" path mean.

Reply:
plymouth.enable=0 only disable certain service from starting up, which boosts system boot speed.

Reply:
sorry, meant "double spaces"

Reply:
it's nice that plymouth.enable=0 boosts system boot speed but shouldn't we test default systems? Anyway, that's not a concern for this PR as this was there already in before

Reply:
PXE boot uses text mode. So this plymouth even outside of default scope.
https://en.wikipedia.org/wiki/Plymouth_(software)

Reply:
> sorry, meant "double spaces"
How I can say. Even "double spaces" better than "no space". But "double" space only occurs to very limited usage scenarios.